playground/mocks/idp/idx/data/authenticator-verification-webauthn.json

```diff
@@ -139,40 +138,52 @@
       }
     ]
   },
-  "authenticators": {
+  "authenticatorEnrollments": {
     "type": "array",
     "value": [
       {
-        "authenticatorType": "password",
-        "authenticatorId": "autwa6eD9o02iBbtv0g3",
-        "authenticatorEnrollmentId": "aenwboITrg4b4yAYd0g3",
-        "authenticatorName": "Okta Password"
+        "label": "aaa",
+        "value": {
+          "type": "password",
+          "authenticatorId": "autwa6eD9o02iBbtv0g1",
+          "id": "aidwboITrg4b4yAYd0g3",
+          "name": "Okta Password"
+        }
       },
       {
-        "authenticatorType": "webauthn",
-        "authenticatorId": "aidtheidkwh282hv8g3",
-        "authenticatorName": "Security Key or Biometric Authenticator (FIDO2)"
+        "label": "bbb",
+        "value": {
+          "type": "webauthn",
+          "authenticatorId": "autwa6eD9o02iBbtv0g2",
+          "id": "aidtheidkwh282hv8g3",
+          "name": "Security Key or Biometric Authenticator (FIDO2)"
+        }
       },
       {
-        "authenticatorType": "email",
-        "authenticatorId": "aidtm56L8gXXHI1SD0g3",
-        "authenticatorName": "Okta Email",
-        "methods": [
-          {
-            "methodType": "email"
-          }
-        ]
+        "label": "ccc",
+        "value": {
+          "type": "email",
+          "id": "aidtm56L8gXXHI1SD0g3",
+          "authenticatorId": "autwa6eD9o02iBbtv0g3",
+          "name": "Okta Email",
+          "methods": [
+            {
+              "methodType": "email"
+            }
+          ]
+        }
       }
     ]
   },
-  "authenticator": {
+  "currentAuthenticator": {
     "type": "object",
+    "label": "bbb",
     "value": {
       "profile": {
         "name": "Security Key or Biometric Authenticator (FIDO2)"
```

Comment:
name is label here as well

Reply:
It's `name` from [this wiki](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/986023151/WebAuthn+End+User+Enrollment+Verification+API+Design+on+IDX+with+Authenticator+Model).
Did you confirm with Michael or Sam already?

Reply:
@haishengwu-okta name shouldnt be there. Check with sam since Tom Abott is also working with him to finalize stuff. LEts treat Sam's wiki as source of truth since the factor wikis might be out of date/